tools/java/org/unicode/cldr/tool/GenerateLanguageMatches.java

```diff
@@ -0,0 +1,105 @@
+package org.unicode.cldr.tool;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
```

Comment:
Optional: I find it clearer not to import Map.Entry but rather to use Map.Entry in the code.

Reply:
That's a toss up; again, we do that all over.